backend/api/Controllers/Models/CreateRobotQuery.cs

```diff
@@ -23,5 +23,7 @@ public struct CreateRobotQuery
         public bool Enabled { get; set; }
 
         public RobotStatus Status { get; set; }
+
+        public AssetDeck CurrentAssetDeck { get; set; }
```

Comment:
One of them is for the asset and the other for the assetdeck ? Forexample look at flotilla\backend\api\Database\Models\Robot.cs you will find the place where they are both used at line 101 and 102

Reply:
`CurrentAsset` stores the same information as `AssetDeck.AssetCode` so I think we can remove it from both this file, and where it is used in `Robot.cs`. And elsewhere in the code, we update to use `Robot.CurrentAssetDeck.AssetCode` instead of `Robot.CurrentAsset`